modules/vcf2maf/1.0/config/default.yaml

```diff
@@ -0,0 +1,21 @@
+lcr-modules:
+    
+    vcf2maf:
+        inputs:
+            # Available wildcards: {out_dir}, {seq_type}, {genome_build}, {tumour_id}, {normal_id}, {pair_status}, {vcf_name}
+            vep_cache: "__UPDATE__"
+
+        options:
+            vcf2maf: "--filter-vcf 0"
+            # --filter-vcf     A VCF for FILTER tag common_variant. Set to 0 to disable [~/.vep/ExAC_nonTCGA.r0.3.1.sites.vep.vcf.gz]
+            #--species        Ensembl-friendly name of species (e.g. mus_musculus for mouse) [homo_sapiens]
+            #--cache-version  Version of offline cache to use with VEP (e.g. 75, 84, 91) [Default: Installed version]
+
+        conda_envs:
+            vcf2maf: "envs/vcf2maf-1.6.18.yaml"
+            
+        threads:
+            vcf2maf: 2
```

Comment:
Any reason not to set this higher by default? I usually use at least 12 threads.

Reply:
No I'm working with small datasets so I thought setting it to 12 threads would increase the wait time for resources. I can change the default to 12 in the final commit

Reply:
Reminder to update the thread number